rollup.config.js

```diff
@@ -20,45 +22,102 @@ const babelOptions = require('./.storybook/.babelrc'); // get the babelrc file
 
 const input = glob.sync('src/components/@(10-atoms|20-molecules|30-organism)/**/index*.js');
 
+const LIB = 'lib';
+const DIST = 'dist';
+
 /* if you want to test lerna publish with demo-button and demo-link
 const input = glob.sync('src/demo/@(demo-button|demo-link)/index*.js');
 */
 
-const plugins = [
-  replace({
-    'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
-  }),
-  sass({
-    insert: true,
-    include: ['**/*.scss'],
-    options: {
-      includePaths: ['node_modules', path.resolve(path.dirname(require.resolve('breakpoint-sass/package.json')), 'stylesheets')],
-      data: globals,
-    },
-    processor: css => postcss([autoprefixer, stripFontFace])
-      .process(css)
-      .then(result => result.css),
-  }),
-  babel({
-    ...babelOptions,
-    babelrc: false,
-    exclude: [
-      'node_modules/**',
-    ],
-    runtimeHelpers: true,
-  }),
-  resolve({
-    jsnext: true,
-    module: true,
-    only: [/^\.{0,2}\/|\.scss$/i], // threat all node_modules as external apart od .scss files
-  }),
-];
+const generatePlugins = type => {
+  const commons = [
+    replace({
+      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+    }),
+    sass({
+      insert: true,
+      include: ['**/*.scss'],
+      options: {
+        includePaths: ['node_modules', path.resolve(path.dirname(require.resolve('breakpoint-sass/package.json')), 'stylesheets')],
+        data: globals,
+      },
+      processor: css => postcss([autoprefixer, stripFontFace])
+        .process(css)
+        .then(result => result.css),
+    }),
+  ];
+  // is LIB
+  if (type === LIB) {
+    return [
+      ...commons,
+      babel({
+        ...babelOptions,
+        babelrc: false,
+        exclude: [
+          'node_modules/**',
+        ],
+        runtimeHelpers: true,
+      }),
+      resolve({
+        jsnext: true,
+        module: true,
+        only: [/^\.{0,2}\/|\.scss$/i], // threat all node_modules as external apart od .scss files
+      }),
+    ];
+  }
+  // is DIST
+  return [
+    ...commons,
+    babel({
+      presets: babelOptions.presets,
+      plugins: [
+        ...babelOptions.plugins,
+        '@babel/plugin-transform-runtime'
+      ],
+      babelrc: false,
+      exclude: [
+        'node_modules/@skatejs/val/**',
+        'node_modules/@babel/**',
```

Comment:
Why are only those two node_modules folders ignored?

Reply:
@codepleb  Cause they dont provide correctly UMD named exports and generate a warning in rollup build. The warning says that those modules are ignored and threated as external. This settings is only here to suppress that warning